### PR TITLE
Add jina-hubble-sdk

### PR DESCRIPTION
**Pull request by floralatin**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by floralatin** [comment]
@conda-forge-admin  what should i do for the GitHub action?

**Comment by floralatin** [comment]
@conda-forge-admin  Could you please help me merge it as soon as, thanks very much.

File: recipes/jina-hubble-sdk/meta.yaml
Comment (dopplershift) [inline recipes/jina-hubble-sdk/meta.yaml:2]: Why such an old version? The latest is 0.23.

File: recipes/jina-hubble-sdk/meta.yaml
Comment (dopplershift) [inline recipes/jina-hubble-sdk/meta.yaml:26]: Do these really need to be pinned exactly to these versions? This usually eventually leads to problems resolving environments that install this package alongside others.

File: recipes/jina-hubble-sdk/meta.yaml
Comment (floralatin) [reply recipes/jina-hubble-sdk/meta.yaml:2]: Our repo `Jina` has dependencies on the old `jina-hubble-sdk` version, so how do we release the old `jina-hubble-sdk `version.

File: recipes/jina-hubble-sdk/meta.yaml
Comment (floralatin) [reply recipes/jina-hubble-sdk/meta.yaml:26]: we use `grayskull pypi --strict-conda-forge` jina-hubble-sdk`` generate the `meta.yaml`. and our repo ` jina-hubble-sdk`'s [requirements]( https://github.com/jina-ai/jina-hubble-sdk/blob/v0.2.0/requirements.txt#:~:text=requests%3D%3D2.27.1,aiohttp%3D%3D3.8.1)

File: recipes/jina-hubble-sdk/meta.yaml
Comment (dopplershift) [reply recipes/jina-hubble-sdk/meta.yaml:2]: Merging this PR will create the initial feedstock with 0.2.0 and packages will be built for it. After that, our auto-tick bot will issue a PR updating to the latest version.

Comment (floralatin) [reply recipes/jina-hubble-sdk/meta.yaml:2]: Thanks for your reply, and please merge this PR if has no other problem.

Comment (floralatin) [reply recipes/jina-hubble-sdk/meta.yaml:2]: > > our auto-tick bot will issue a PR updating to the latest version.

 What should I do if I want to publish all the old versions before (according to the version on pypi)？

File: recipes/jina-hubble-sdk/meta.yaml
Comment (floralatin) [reply recipes/jina-hubble-sdk/meta.yaml:26]: > Do these really need to be pinned exactly to these versions?

Yes, keep it consistent with the git version.

File: recipes/jina-hubble-sdk/meta.yaml
Comment (dopplershift) [reply recipes/jina-hubble-sdk/meta.yaml:2]: You can make manual PRs to the main branch updating to each version you want and avoid merging the automated PR until you're done getting older versions.

With that said, I'd strongly encourage you to consider whether you actually need all of those old versions.

Comment (floralatin) [reply recipes/jina-hubble-sdk/meta.yaml:2]: Yes, we actually need all of those old versions. thanks for your reply.

File: recipes/jina-hubble-sdk/LICENSE
Comment (dopplershift) [inline recipes/jina-hubble-sdk/LICENSE:1]: I don't see this license anywhere upstream. We can't just add a new license here, it needs to come from upstream. The link to the Apache-2.0 license upstream redirects to a 404 for a non-existent license file, while the license metadata in the upstream `setup.py` says "Proprietary".

We need an accurate license file actually endorsed by upstream in order to know whether we can legally redistribute this package.

File: recipes/jina-hubble-sdk/meta.yaml
Comment (dopplershift) [inline recipes/jina-hubble-sdk/meta.yaml:50]: I need confirmation from @mapleeit @nomagick @delgermurun and @Andrei997 that they are willing to be listed as maintainers for this recipe.

Comment (floralatin) [reply recipes/jina-hubble-sdk/meta.yaml:50]: > I need confirmation from @mapleeit @nomagick @delgermurun and @Andrei997 

They are my colleagues.

File: recipes/jina-hubble-sdk/LICENSE
Comment (floralatin) [reply recipes/jina-hubble-sdk/LICENSE:1]: We have add add accurate [license file](https://github.com/jina-ai/jina-hubble-sdk/blob/main/LICENSE) in our rep `jina-hubble-sdk`  but It's from version v0.24.0, is there a problem.

File: recipes/jina-hubble-sdk/meta.yaml
Comment (delgermurun) [reply recipes/jina-hubble-sdk/meta.yaml:50]: Yes, I can confirm. We're colleagues and maintainers of https://github.com/jina-ai/jina-hubble-sdk.
Thank you.

Comment (Andrei997) [reply recipes/jina-hubble-sdk/meta.yaml:50]: Yes, I can confirm as well.

Comment (dopplershift) [reply recipes/jina-hubble-sdk/meta.yaml:50]: > They are my colleagues.

Because these individuals will be sent invites to the repository and potentially more GitHub notifications we are obligated to make sure everyone approves.

Comment (nomagick) [reply recipes/jina-hubble-sdk/meta.yaml:50]: Yes. I also confirm. Thank you.

**Review by delgermurun — APPROVED** [review]

File: recipes/jina-hubble-sdk/meta.yaml
Comment (mapleeit) [reply recipes/jina-hubble-sdk/meta.yaml:50]: Yes. Confirm. Thanks.

**Review by dopplershift — APPROVED** [review]

**Pull request merged**
